Sprint-2/5-stretch-extend/format-time.js

```diff
@@ -7,7 +7,12 @@ function formatAs12HourClock(time) {
   if (hours > 12) {
     return `${hours - 12}:00 pm`;
   }
+  else if (hours === 12){
+    return `${hours}:00 pm`;
+  }
+  else {
   return `${time} am`;
+  }
 }
```

Comment:
what if time is 24:00 
`const newTime = formatAs12HourClock("24:00");`
 can you check your fn again to come up with correct  solution